src/thread_dispatcher.cpp

```
@@ -122,12 +122,43 @@ static void resume_stopwatch(Utils::StopWatch& s, const std::string& reason)
 }
 // LCOV_EXCL_STOP
 
+static void dump_message_details(pjsip_rx_data* rdata)
+{
+  if (rdata != NULL)
```

Bit weird that a function that takes an rdata and operates on an rdata silently does nothing if an rdata isn't passed in.